notebooks/lesson14_image-dataset.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>replace "I" with "we"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>